### ISSUE TITLE
php_version missing 8.3

### ISSUE THREAD

**Issue by ondruska** [hashicorp/terraform-provider-azurerm#26121]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Terraform Version

1.8.3

### AzureRM Provider Version

3.105.0

### Affected Resource(s)/Data Source(s)

azurerm_linux_web_app

### Terraform Configuration Files

```hcl
resource "azurerm_linux_web_app" "webapp" {
  name                = "webapp-php-83"
  resource_group_name = data.azurerm_resource_group.rg.name
  location            = azurerm_service_plan.asp.location
  service_plan_id     = azurerm_service_plan.asp.id
  site_config {
    application_stack {
      php_version = "8.3"
    }
  }
}
```

### Debug Output/Panic Output

```shell
sorry, cannot, reveals sensitive information
```

### Expected Behaviour

Create azurerm_linux_web_app with PHP 8.3 runtime

PHP version 8.3 is valid for Azure App Service with PHP runtime:
PS> az webapp list-runtimes --os linux
...
  "PHP:8.3",
  "PHP:8.2",
  "PHP:8.1",
  "PHP:8.0",
...

### Actual Behaviour

Plan: 0 to add, 6 to change, 0 to destroy.
╷
│ Error: expected site_config.0.application_stack.0.php_version to be one of ["7.4" "8.0" "8.1" "8.2"], got 8.3
│
│   with azurerm_linux_web_app.webapp,
│   on app.tf line 34, in resource "azurerm_linux_web_app" "webapp":
│   34:       php_version = "8.3"
│
╵

**Comment by Chambras** [comment]
Hi @ondruska
good catch
I will take a look and try to submit PR soon.

**Comment by Chambras** [comment]
Hi @ondruska PR was submitted. Hopefully will be reviewed and merged soon and released in 3.107.0 maybe?
